pkg/loghttp/push/push.go

```diff
@@ -16,17 +16,19 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"github.com/go-kit/log"
+	"github.com/gogo/protobuf/proto"
```

Comment:
can this be gogo proto? i am not sure how different this is from the google version

Reply:
Unfortunately not. I tried but panics.